slides/kube/daemonset.md

```diff
@@ -503,7 +508,7 @@ rng-xbpvg                   1/1       Running       0          7m
 
 --
 
-- We should see deployment "rng" is deleted. Let's check:
+- The pod that was created by the deployment is now being shutdown:
```

Comment:
"shutdown" which should be "shut down" feels kinda ambiguous. Terminated, maybe? I don't think "shutting down" is what I think a pod does?

Reply:
Yup you're right! Since the command shows it in `Terminating` state, I guess we should just say it's "now being terminated."